modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -89,12 +89,13 @@ The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate
 
 `ResourceDeletion`:: This remediation strategy removes the pods and associated volume attachments on the node rather than the node object. This strategy helps to recover workloads faster. `ResourceDeletion` is the default remediation strategy.
 
-`NodeDeletion`:: This remediation strategy removes the node object.
+`NodeDeletion`:: This remediation strategy is deprecated and will be removed in a future release. In the current release, the `ResourceDeletion` strategy will be used even if the `NodeDeletion` strategy is selected.
 
-The Self Node Remediation Operator creates the following `SelfNodeRemediationTemplate` CRs for each strategy:
+
+The Self Node Remediation Operator creates the following `SelfNodeRemediationTemplate` CR for the strategy:
 
 * `self-node-remediation-resource-deletion-template`, which the `ResourceDeletion` remediation strategy uses
-* `self-node-remediation-node-deletion-template`, which the `NodeDeletion` remediation strategy uses
+//* `self-node-remediation-node-deletion-template`, which the `NodeDeletion` remediation strategy uses
```

Comment:
> This looks really good! I had some suggestions but nothing major.

Thank you for taking the time to review Kerry. Much appreciated.

Reply:
> Since this one is commented out, do you want to keep a bulleted list if there is only one item in it? If there will be more eventually maybe it makes sense to keep it this way.

Yes, only one item for this release, but kept this structure in place as there will be more items in the next release.